src/metrics/chaoss.ts

```diff
@@ -305,6 +305,70 @@ FORMAT JSONCompact`;
   });
 };
 
+interface IssueResponseTimeOptions {
+  thresholds: number[];
+  unit: 'month' | 'week' | 'day' | 'hour' | 'minute';
+}
+export const chaossIssueResponseTime = async (config: QueryConfig<IssueResponseTimeOptions>) => {
+  config = getMergedConfig(config);
+  const whereClauses: string[] = ["type IN ('IssueCommentEvent', 'IssuesEvent') AND actor_login NOT LIKE '%[bot]'  "];
+  const repoWhereClause = await getRepoWhereClauseForClickhouse(config);
+  if (repoWhereClause) whereClauses.push(repoWhereClause);
+  const endDate = new Date(`${config.endYear}-${config.endMonth}-1`);
+  endDate.setMonth(config.endMonth);  // find next month  
+  let thresholds = config.options?.thresholds ?? [1, 3, 7]; // 3 thredholds will separate the value to 4 ranges
```

Comment:
Add `thresholds` and `unit` options, default is <1 day, 1-3 days, 3-7 days and > 7 days. Users can customize their own options like [12, 24, 36, 48] hours.